### ISSUE TITLE
new warp system makes larger ships obsolete

### ISSUE THREAD

**Issue by king32109** [tsunamayo/Starship-EVO#5869]
### Short Description of the issue:

i love the new system let me say that for warp, i love the use of void pearls to make them able to jump, but i would personaly like to see it changed a tiny bit to make ships that are alot larger able to warp with less pearls, a ship i am building (300m long) that is only hull for now needs 139 pearls to jump once, and i find that to much, i would have to spend more time looking for pears then actully playing the game for it to jump once to another system, wich makes this well not fun to do or play the game, this system while unque and fun leavers small ships in a clear advantage then ships of larger size, 

### Environment
- Starship EVO version: 24w28a
- Graphics Card: 

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.

**Comment by king32109** [comment]
the ship weighs 7000 tonnes, but to add to what you said about not builind that big in survival, if you dont want that, then why have the scaling system that goes to 32m large for a single block feels a bit limiting to keep builds small just for the case warp when you can build stuff thats con be a km long or bigger

**Comment by Crimson-Artist** [comment]
The solution is simple: add a new type of void pearl called a "Super Void Pearl" or something like that that is designed to allow larger ships to warp. It'll be rarer than regular void pearls so large ships will still be harder to use but it'll be valuable and maybe an end game resource

**Comment by asanagisae** [comment]
What about a new manufacturing block type that takes 100 void pearls and compresses them into a super void pearl? Small pearls aren't going to be very efficient taking up entire slots in large scale warp containers. By the time you find a super void pearl naturally on a planet or asteroid, you should be able to craft 2 or 3 by combining the little void pearls you picked up along the way.

It would be nice if we could "recycle" partially used void pearls in a similar manner (by compressing them). Instead of just picking the largest void pearls from each cluster we see, we can take them all and compress them into fewer pearls before putting them in the warp cores.

----

Visually the warp pearls scale up, but they still produce the same amount of warp fuel. When you pick them up they have different sizes to signify the fuel they contain, but they lose that distinction when you put them in a warp container.

![image](https://github.com/tsunamayo/Starship-EVO/assets/10069469/c9d628a2-19d1-4725-ba09-f5fe91079f51)

![image](https://github.com/tsunamayo/Starship-EVO/assets/10069469/4b8c23e8-bba1-4a19-a742-4c64bf995ea7)

As it is, larger scale warp containers aren't very space efficient. Each scale up takes 8x the volume but it holds the same amount of fuel. The pearls just _look_ bigger. There's no reason to use the large 4m warp containers at all.

When we get multiplayer servers, perhaps we can add some method for players to sell void pearls to each other? 100 pearls sounds like a big number, but for players specialized in void pearl gathering, that's only 5 minutes of work.

I think the void pearl system is alright.

**Comment by ProPeach** [comment]
Awww, I had hoped that the larger scale pearl storage bricks would look more like this 😄 
![image](https://github.com/user-attachments/assets/5caf3706-e60e-4478-a1bf-e57bf6104028)
It would make a lot more sense for the pearls to keep a consistent size even across storage brick scales, it would give you an instant idea of how many you have in there, and how many you have left

**Comment by TIKIRobo** [comment]
https://youtu.be/egTtyS-PlRM?si=IA6XPIPo_meOme92&t=41
hear me out...

i think it would be really cool, and funny, if the pearls were stored like this (floating around in a larger glass tank)
and maybe the storage unit would use more power per stored pearl?

**Comment by tsunamayo** [comment]
Yes I agree on the scale thing, I was really close to enforce either (or both) a single pearl scale or a single grid scale for warp core. I am not coding a system that adapt with scale. This is where creative freedom just gets in the way a of survival dev and also ease of use for the player.
I did not wanted to put a limit on the scale of pearl you can store so that it is less frustrating for the player. 
But I will apply a different scaling, so smaller pearl looks a bit smaller.

I like the pearl merging thing, I think that would be the best way to proceed.

@king32109 I think your math is off as I suspected, 7000T will require 140 0.5m pearl for _10x_ jumps, not one (or there is a bug). 
Right now a 300T ship will require 6 0.5m pearl to do 10x jumps. That seems very reasonable to me, 300T is already quite big, and 6 pearl is not a lot.
Having to spend some extra time every 10x doesnt seems absurd to me, if you build carrier size in survival well you have to own to it and spend a bit of time for refueling. And it wont take you 1h to get a 100 peal I think, with a ship and a collector it should be fast.

**Comment by asanagisae** [comment]
I believe the original reaction was "138 (one hundred and thirty eight) void pearls? Collecting _over a hundred_ void pearls for a single warp jump is too much", and the confusion is that the estimate is listed for "10 jumps" (not one jump). If it read "13,888 for _one_ jump", it would be less of a shock.

Unless you're thinking something like "13 void pearls per jump, or buy 10 jumps and get one free"? 😄

**Comment by tsunamayo** [comment]
@ProPeach I use standard .net function that take care of the formatting depending on the local of your windows.
https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings
So everybody sees what make sense in his country.

For the rest please look into the other post #5865 for what I plan to do. It make more sense to have the conversation at a single place, but it address the size issue @ProPeach 
Black pearl are pearl that are fully discharged. I prefer that over to pearl size reduce with fuel left, although that would be also an option that would solve the disposal of discharged pearl.

**Comment by Crimson-Artist** [comment]
Drained pearls could be used a crafting ingredient for something. I had the idea that maybe the pearls are explosive and could be the basis of explosive weapons like missiles, mines, or a dedicated explosive block like TNT in minecraft. fresh pearls would be preferred so you don't have to warp around just to gain access to explosive weapons. Maybe it will take more drain pearls to produce the same explosive component. for instance the explosive component has two crafting recipes: the first requiring only one fresh pearl and the other requires 5 drained pearls.

**Comment by Ketsueko** [comment]
to me depleted pearl could have a few use like uranium: a type of highly efficient fuel, a type of armor that better diffuse heat, or a type of ammunition/beam/laser augmentation that could maybe be some sort of "MK2 weapon module"? Personnaly i would love a increase range/velocity, reduce spread/ increase damage at the cost of longer charge time/ heat generation (like that, even beam weapon would have longer range than 2km)

**Comment by vortenian** [comment]
late game way to recharge used void perils with dangerous proximity to a star? your ship would need to be able to take on heat damage and strong enough to pull out of the gravity or else failure. would be neat. something to think about post release maybe. 

for now i would keep pushing forward.
